libraries/AP_Terrain/AP_Terrain.cpp

```diff
@@ -309,12 +304,9 @@ void AP_Terrain::update(void)
     // just schedule any needed disk IO
     schedule_disk_io();
 
-    // try to ensure the home location is populated
-    float height;
-    height_amsl(ahrs.get_home(), height, false);
```

Comment:
Although we don't use the return value, I think this height_amsl of the home location is what we want.  It ensures that we fetch the terrain height of home.